### PR TITLE
cgdb: Add texinfo depend

### PR DESCRIPTION
**Pull request by t-nojiri**
I have fixed the following issues:
>> 137    configure: error: Please install makeinfo before installing

### CODE REVIEW

**Review by alalazo — CHANGES_REQUESTED** [review]
Can you please double check this package? The installation instructions from [their repository](https://github.com/cgdb/cgdb) seem to point to a lot of missing dependencies. I wonder if right now they're found on the system.

**Comment by t-nojiri** [comment]
I have confirmed that all the commands listed in Dependencies are installed on my system.

**Comment by alalazo** [comment]
@t-nojiri Can you fix those dependencies as well in this PR?

**Comment by t-nojiri** [comment]
I added depend.

File: var/spack/repos/builtin/packages/cgdb/package.py [hunk 1]
```diff
@@ -17,6 +17,12 @@ class Cgdb(AutotoolsPackage):
     # Required dependency
     depends_on('ncurses')
     depends_on('readline')
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/cgdb/package.py]: I wonder if these dependencies are actually needed to build from a release, the tarball already contains a configure script.

Comment (t-nojiri) [reply var/spack/repos/builtin/packages/cgdb/package.py]: I was able to read the spack manual and understand what was pointed out.

**Review by adamjstewart — APPROVED** [review]

**Review by alalazo — APPROVED** [review]

**Pull request merged**